### DISCUSSION TITLE
jdbc cant connect MySQL server via vpnhotspot

### DISCUSSION THREAD

**Discussion by yuanli-1** [Mygod/VPNHotspot#595] [Q&A]
jdbc maybe use TLSv1 or other encypt, how can i use jdbc  connect MySQL server via vpnhotspot?
here is the log
[vpnhotspot-5550328841822554556.log](https://github.com/user-attachments/files/16401424/vpnhotspot-5550328841822554556.log)

**Comment by Mygod** [comment]
All connections should work as usual.
